src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipPatchMapperTest.java

```diff
@@ -39,12 +40,17 @@ void testObjectMapperCanHandleJsonNullableFields() throws JsonProcessingExceptio
 
     @ParameterizedTest
     @CsvSource(value = {
-            // Field NOT present in the JSON - no update:
-            "{\"email\":\"test@test.com\"}$ Asset Strippers $ test@test.com",
-            // Field IS present in the JSON - set the new string value:
-            "{\"partnership_name\":\"Asset Adders\", \"email\":\"test@test.com\"}$ Asset Adders $ test@test.com"
+            // Fields NOT present in the JSON - no update:
+            "{\"email\":\"test@test.com\"}$ Asset Strippers $ test@test.com $ Scotland",
```

Comment:
Is it possible to put these literals into variables?

Reply:
Done.